roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -45,37 +45,45 @@ host = ceph
 glance_host = {{ endpoints.main }}
 glance_port = {{ endpoints.glance.port.backend_api }}
 
-{% if cinder.enabled_backends|length -%}
-enabled_backends = {{ cinder.enabled_backends|join(',') }}
+{% if ceph_pools.rbd_ssd.enabled and ceph_pools.rbd_hybrid.enabled -%}
```

Comment:
`ceph_pools` is set in `ceph-osd/tasks/pool_names.yml`. yes, this variable may not exist if ceph-osd role not executed. Considering the case of ceph disabled, we can make ceph related logic executed in `if ceph.enabled`, but never set default value of `ceph_pool`, this variable should always be set according to current cluster.